plugins/environment.go

```diff
@@ -115,11 +116,21 @@ When the -plugin option is specified, these flags are ignored.`)
 		err = proto.Unmarshal(apiData, documentv3)
 		if err == nil {
 			env.Request.AddModel("openapi.v3.Document", documentv3)
+			sourceName := strings.Replace(*input, ".pb", ".yaml", -1)
+			if _, err := os.Stat(sourceName); os.IsNotExist(err) {
+				// sourceName does not exist. Lets try .json instead
+				sourceName = strings.Replace(*input, ".pb", ".json", -1)
```

Comment:
I'm not clear on how this code is getting exercised. Referring back to line 37/38, it says that we expect the input to be in binary form. Are we now also allowing JSON/YAML input?

Reply:
We still expect the input to be in binary form. This is the same sourceName as in the [gnostic.go](https://github.com/googleapis/gnostic/pull/135/files#diff-07b5e04f760e7b7bac224fec4ccf7f7cR158) file. However, in the case above we run the plugin as command line tool with a binary file as input. So our best bet is to guess the sourceName from which the binary file is coming from.